src/test/java/org/elasticsearch/indices/analyze/AnalyzeActionTests.java

```diff
@@ -259,4 +259,47 @@ public void testParseXContentForAnalyzeRequestWithUnknownParamThrowsException()
         }
     }
 
+    @Test
+    public void analyzerWithMultiValues() throws Exception {
+
+        assertAcked(prepareCreate("test").addAlias(new Alias("alias")));
+        ensureGreen();
+
+        client().admin().indices().preparePutMapping("test")
+            .setType("document").setSource(
```

Comment:
.setSource("simple", "type=string,analyzer=simple,position_offset_gap=100") ? so we can remove json provided as a string?